raiden/messages.py

```diff
@@ -1440,6 +1443,143 @@ def from_dict(cls, data):
+
+    def __init__(
+            self,
+            chain_id: ChainID,
+            nonce: int,
+            message_identifier: MessageID,
+            transferred_amount: TokenAmount,
+            locked_amount: TokenAmount,
+            locksroot: Locksroot,
+            channel_identifier: ChannelID,
+            token_network_address: TokenNetworkAddress,
+            recipient: Address,
+            secrethash: SecretHash,
+    ):
+
+        super().__init__(
+            chain_id=chain_id,
+            nonce=nonce,
+            transferred_amount=transferred_amount,
+            locked_amount=locked_amount,
+            locksroot=locksroot,
+            channel_identifier=channel_identifier,
+            token_network_address=token_network_address,
+        )
+        self.message_identifier = message_identifier
+        self.recipient = recipient  #: partner's address
+        self.secrethash = secrethash
+
+    @classmethod
+    def unpack(cls, packed):
+        transfer = cls(
+            chain_id=packed.chain_id,
+            nonce=packed.nonce,
+            message_identifier=packed.message_identifier,
+            token_network_address=packed.token_network_address,
+            channel_identifier=packed.channel_identifier,
+            transferred_amount=packed.transferred_amount,
+            recipient=packed.recipient,
+            locked_amount=packed.locked_amount,
+            locksroot=packed.locksroot,
+            secrethash=packed.secrethash,
+        )
+        transfer.signature = packed.signature
+
+        return transfer
+
+    def pack(self, packed):
+        packed.chain_id = self.chain_id
+        packed.nonce = self.nonce
+        packed.message_identifier = self.message_identifier
+        packed.token_network_address = self.token_network_address
+        packed.channel_identifier = self.channel_identifier
+        packed.transferred_amount = self.transferred_amount
+        packed.locked_amount = self.locked_amount
+        packed.recipient = self.recipient
+        packed.locksroot = self.locksroot
+        packed.secrethash = self.secrethash
+        packed.signature = self.signature
+
+    @classmethod
+    def from_event(cls, event):
+        balance_proof = event.balance_proof
+
+        return cls(
+            chain_id=balance_proof.chain_id,
+            nonce=balance_proof.nonce,
+            token_network_address=balance_proof.token_network_identifier,
+            channel_identifier=balance_proof.channel_identifier,
+            transferred_amount=balance_proof.transferred_amount,
+            locked_amount=balance_proof.locked_amount,
+            locksroot=balance_proof.locksroot,
+            message_identifier=event.message_identifier,
+            recipient=event.recipient,
+            secrethash=event.secrethash,
+        )
+
+    def __repr__(self):
+        representation = (
+            '<{} ['
+            'chainid:{} token_network:{} msg:{} channel_identifier:{} nonce:{} '
```

Comment:
token_network -> token_network_address
msg -> msg_id

Just wanting to be a bit more explicit